src/components/Footer/Fragments/LinkGrid.js

```diff
@@ -1,23 +1,31 @@
-import { Box, Link, SimpleGrid, Stack } from "@chakra-ui/react"
+import { Box, SimpleGrid, Stack, Link } from "@chakra-ui/react"
+import { Link as ReachLink } from "@reach/router"
 import * as React from "react"
 import { FooterHeading } from "./FooterHeading"
 
 const LinkGrid = props => (
-  <SimpleGrid columns={2} {...props}>
-    <Box minW="130px">
-      <FooterHeading mb="4">Product</FooterHeading>
-      <Stack>
-        <Link>How it works</Link>
-        <Link>Pricing</Link>
-        <Link>Use Cases</Link>
-      </Stack>
+  <SimpleGrid columns={{base: 1, lg: 2}} {...props}>
+    <Box minW="140px">
+      <FooterHeading mb="4">Menu</FooterHeading>
+      <SimpleGrid columns={{base: 2, lg: 2}}>
+        <Box mr="5">
+          <Link as={ReachLink} to="/about/">About the Artist</Link>
+          <br />
```

Comment:
Shouldn't use the `<br/>` tags, try making the container display block or flex-direction column